modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointPagesWriter.java

```diff
@@ -117,7 +120,7 @@ public class CheckpointPagesWriter implements Runnable {
             IgniteLogger log,
             CheckpointMetricsTracker tracker,
             IgniteConcurrentMultiPairQueue<PersistentPageMemory, FullPageId> writePageIds,
-            IgniteConcurrentMultiPairQueue<PersistentPageMemory, GroupPartitionId> updatePartitionIds,
+            Set<GroupPartitionId> savedPartitionMetas,
```

Comment:
Interesting, why does it have to be a parameter? I don't think I get it, are we use this collection somewhere outside of the writer?

Reply:
So that all writers (this is several threads) do not write the same meta.

Reply:
Ok, makes sense now, thank you!